src/genome.html

```diff
@@ -91,6 +94,32 @@ <h2>Genome <% print(accession) %></h2>
             <p>A full copy of the raw analysis data is available in the <a id="downloads-link-bottom" href="javascript:void(0);">downloads tab</a>.</p>
         </div>
 
+        <script id="cataloguesTmpl" type="text/template">
```

Comment:
Sorry a bit confused... where is this used? I mean I see it been used in `genome.js`, but I'm still haven't found it while browsing in the website.

Reply:
Great catch. This was relevant to a previous iteration where we showed links to all the catalogues in which a genome was present. This is now one-to-one so no longer used. Deleting.